src/libcadet/model/reaction/MassActionLawReaction.cpp

```diff
@@ -26,6 +26,9 @@
 #include <string>
 #include <vector>
 
+#include <Eigen/Dense>
```

Comment:
I think its time to make Eigen obligatory for CADET.
Currently, Eigen is only installed when the `ENABLE_DG` flag is set to ON (which is the default)
So when you are using Eigen in another submodule, the build will fail when `ENABLE_DG` is OFF

We can either make Eigen obligatory for CADET or you also guard the rapid-equilibrium reaction code with compile time flags checking for Eigen